platform/centec/rules.mk

```diff
@@ -1,5 +1,5 @@
+include $(PLATFORM_PATH)/platform-modules-centec-e582.mk
 include $(PLATFORM_PATH)/sdk.mk
-include $(PLATFORM_PATH)/sai.mk
```

Comment:
if sai.mk file is no longer needed, can you remove that file?

Reply:
thanks, i will close this PR and create a new PR after these changes are done.